netmanager/src/App.js

```diff
@@ -102,65 +100,56 @@ const App = () => {
               />
```

Comment:
Side note: we shall need to transfer this route logic to the routers directory accordingly. Thanks.

Reply:
This component holds these high level components (components that require the layout component). The routes are quite many but removing them will leave this component empty or unnecessary. Not sure which approach is best here but leaving as it is may be the best.

Reply:
Cool, thanks.